plexpy/activity_handler.py

```diff
@@ -251,6 +253,10 @@ def process(self):
                         if int(time.time()) - db_session['stopped'] > 60:
                             self.update_db_session()
 
+                    # Update db session when view offset changes
+                    if this_view_offset != last_view_offset:
+                        self.update_db_session()
```

Comment:
I don't like updating the database for every single websocket event. This is why I have the line right above to update every 60 seconds. I think it is acceptable for the currently active items in the history table to be at most 60 seconds out. They will be correct when the stream is stopped anyways.

Reply:
Oh, okay! I'll revert this change then.